libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -698,3 +704,18 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
         list-style: none;
     }
 }
+
+button.gd-button-primary {
+    &.icon-sources.s-sources.gd-button {
+        > span {
+            margin-left: 20px;
```

Comment:
![image](https://user-images.githubusercontent.com/67769750/103207454-f144ae00-4930-11eb-90b9-b531bee031ca.png)

Is `margin-left` here the margin from the text `Sources` to the left border of their containers? If that's right, please change the value to that used in the mockup.

Reply:
it's a distances from the left side of icon to text. But thanks i have updated the exactly px need to fix with it